.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,16 +5,28 @@ The more detailed you are, the better.**
 
 Before submitting the PR make sure the following are checked:
 
-* [ ] The PR relates to *only* one subject with a clear title and description in grammatically correct, complete sentences.
-* [ ] Wrote [good commit messages][1].
-* [ ] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
+* [ ] The PR relates to _only_ one subject with a clear title and description in grammatically correct, complete sentences.
 * [ ] Feature branch is up-to-date with `master` (if not - rebase it).
 * [ ] Squashed related commits together.
 * [ ] Added tests.
-* [ ] Ran `bundle exec rake default`. It executes all tests and runs RuboCop on its own code.
+* [ ] Updated documentation.
 * [ ] Added an entry (file) to the [changelog folder](https://github.com/rubocop/rubocop-rails/blob/master/changelog/) named `{change_type}_{change_description}.md` if the new code introduces user-observable changes. See [changelog entry format](https://github.com/rubocop/rubocop/blob/master/CONTRIBUTING.md#changelog-entry-format) for details.
-* [ ] The PR relates to *only* one subject with a clear title
-  and description in grammatically correct, complete sentences.
-* [ ] If this is a new cop, consider making a corresponding update to the [Rails Style Guide](https://github.com/rubocop/rails-style-guide).
+* [ ] The build (`bundle exec rake`) passes (be sure to run this locally, since it may produce updated documentation that you will need to commit).
+* [ ] Wrote [good commit messages][1].
+* [ ] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
+
+If you have created a new cop:
+
+* [ ] The cop has been checked against `real-world-rails`. See [https://github.com/rubocop/rubocop-rails/blob/master/CONTRIBUTING.md#real-world-rails]
```

Comment:
The reference doesn't look right
![image](https://user-images.githubusercontent.com/6916/159028039-5286fa1a-0505-442c-9d94-c5d0bece7876.png)